content/en/blog/2023/announcing-notation-rc6.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Notation v1.0.0-RC.7 is available!
+author:  "Notation Release Team"
+date: 2023-05-28
+draft: false
+---
+
+The Notation maintainers are pleased to announce the release of Notation v1.0.0-RC.7, including [Notation CLI v1.0.0-rc.7](https://github.com/notaryproject/notation/releases/tag/v1.0.0-rc.7), [notation-go library v1.0.0-rc.6](https://github.com/notaryproject/notation-go/releases/tag/v1.0.0-rc.6), and [notation-core-go library v1.0.0-rc.4](https://github.com/notaryproject/notation-core-go/releases/tag/v1.0.0-rc.4). This blog walks you through the updates in this release.
+
+> NOTE: Both Notation CLI v1.0.0-rc.7 and v1.0.0-rc.6 have the same functionalities. However, v1.0.0-rc.7 included an additional fix for an E2E test case.
+
+## What's new
+
+This release adds the following major changes:
+
+- Security advisory fixes
+- Notation commands support reading Docker credentials if the [credentials store](https://docs.docker.com/engine/reference/commandline/login/#configure-the-credentials-store) is not configured
+- Renamed `--plain-http` to `--insecure-registry`, which should be used only for testing
+
+Other changes:
+
+- Improved error messages
+- Bug fixes
+- Updated dependencies
+
+### Notation commands support reading docker credentials if credentials store is not installed
+
+Users may provide credentials to authenticate with OCI-compliant registries. As a security best practice, it is highly recommended that users configure a credentials store to securely store their credentials, especially in production environments. However, there may be cases where a credentials store is not configured, such as in a test environment. If the credentials store is not present, Docker stores the credentials in the config files by default, see [reference](https://docs.docker.com/engine/reference/commandline/login/#default-behavior). With the release of Notation v1.0.0-RC.7, notation commands now support reading credentials from the Docker config file if a credentials store is not installed. If users have successfully logged in using `docker login`, they can use the `notation sign` command to sign container images directly without the need to use `notation login` first. The same solution applies to other commands `notation verify`, `notation ls`, and `notation inspect`. For example,
```

Comment:
My understanding is different. Let me know if you are ok with this instead.

Security best practices recommend that users configure a credentials store to securely manage credentials, especially in production environments. Starting with Notation v1.0.0-RC.7, Notation commands use the following order to find credentials to authenticate with registries. Notation commands first look for credentials in its Notation’s config file, if absent Notation looks for credentials in docker config file, and if even that is not present, Notation uses the operating system default. This way, for users who do not configure explicit credentials in Notation, a successful `docker login`, will enable them to run “notation sign/verify/list/inspect” commands without the need to complete `notation login` first.

Reply:
Need to add the new scenario: if none of credential stores are present, Notation will read credentials from docker config if users `docker login` successfully.